environment_set_up/ManualConfig-HPC.md

```diff
@@ -0,0 +1,60 @@
+# Getting Started with On-Prem HPC
+
+This tutorial covers how to set up and use the Pangeo environment/stack
+on  USGS High Performance Computing (HPC) systems:
+
+1) Creating a custom hytest conda environment
+2) Configuring [Jupyter](https://jupyter.org/) server
+3) Launching a Jupyter server
+4) Connecting to your Jupyter server and the[Dask](https://dask.pydata.org/) 
+   dashboard from your personal computer
+
+This document assumes that you already have an access to
+Denali and Tallgrass and are comfortable using the command line.
+
+## Installing a Conda Software Environment
+
+This is the same manual process as describe in the 
+[Quick-Start for HPC](./QuickStart-HPC.md)
+
+## Configure Jupyter
+
+Jupyter notebook servers include a password for security. First we generate the Jupyter config file then set a password:
+
+```text
+jupyter server --generate-config
+jupyter server password
+```
+
+This creates the file `~/.jupyter/jupyter_server_config.py`.
+
+Finally, we configure dask\'s dashboard to forward through
+Jupyter.  Add the following line to your `~/.bashrc` file:
+
+```sh
+export DASK_DISTRIBUTED__DASHBOARD__LINK="/proxy/8787/status"
+```
+
+## Start a Jupyter Notebook Server
+
+Now that we have Jupyter configured, we can start a notebook server on our interactive compute node. Use the [script provided](./start_jupyter.sh) to do this.
+
+```text
+> salloc -A account_name -t 02:00:00 -N 1 srun --pty bash
+> cd $HOME/HyTEST-Tutorials/
+> ./HPC/start_jupyter.sh
+```
+
+Follow the steps printed out by the script to get connected. Note that after you login using the `ssh` command in Step 2, the cursor will hang.  This is expected and you can minimize the window.
+
+## Running Notebooks
+
+### Special Notes for dask jobqueue
```

Comment:
I haven't used dask-jobqueue before so this is an uninformed question. From reading [Rich's instructions](https://github.com/USGS-python/hytest-development/blob/main/hpc.md#launch-dask-with-dask-jobqueue) it sounds like I have to run this Python code to get started.

I see that you are linking to a whole notebook with instructional material.

How does this code fit in to the workflow?
- Is it included in our notebooks and so telling the user to run this code is not necessary. And your notebook is just an explainer for what dask-jobqueue is?
- Or does this piece of code need to get run just once during setup?

Reply:
The original plan is to point to the jobqueue notebook as an explainer/demo.  That code actually gets executed as part of the `configure_cluster()` function, and would run automatically if a user uses that function.

Reply:
I wonder if we even need this link, actually.  Could we just strike that last sentence and reference to the xx_dask_jobqueue.ipynb notebook. ??

Reply:
Thanks for clarifying - maybe we can move this into a `tutorial` subfolder of `environment_set_up` then and update any links to reflect the new path.

Reply:
Yeah I think we should rephrase this and maybe just have text that says something like "you can check out the [xx_dask-jobqueue.ipynb]() notebook for more info"

Reply:
We use dask-jobqueue when we are on tallgrass, because there are only a few CPUs per node and we are often sharing them with others.

Reply:
I changed the language a little -- and I converted the jobqueue notebook to a flat markdown file.  Seems like that document just contained example code, rather than something that we want a reader to execute within that document.